### PR TITLE
adopt the latest build sdk

### PR DESCRIPTION
**Pull request by northtyphoon**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [ ] Each command and parameter has a meaningful description.
- [ ] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py [hunk 1]
```diff
@@ -84,11 +84,11 @@ def load_arguments(self, _):
...
         c.argument('source_location', options_list=['--context', '-c'], help="The local source code directory path (eg, './src') or the url to a git repository (eg, 'https://github.com/docker/rootfs.git') or a remote tarball (eg, 'http://server/context.tar.gz').")
         c.argument('docker_file_path', options_list=['--file', '-f'], help="The relative path of the the docker file to the source code root folder (Default is 'Dockerfile').")
         c.argument('timeout', help='The build timeout in seconds.')
         c.argument('build_args', nargs='+', help='The space-separated build arguments in a format of <name>=<value>.')
         c.argument('secret_build_args', nargs='+', help='The space-separated secret build arguments in a format of <name>=<value>.')
-        c.argument('show_logs', nargs='?', required=False, const='true', default='true', help='Indicates whether showing logs after successfully queuing the build.', choices=['true', 'false'])        
+        c.argument('no_logs', options_list=['--no_logs'], action='store_true', help='Do not show logs after successfully queuing the build.')
```
Comment (djyou) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py]: `--no-logs`

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py [hunk 2]
```diff
@@ -84,11 +84,11 @@ def load_arguments(self, _):
 
     with self.argument_context('acr build') as c:
         c.argument('build_id', help='The unique build identifier.')
-        c.argument('image_name', options_list=['--tag', '-t'], help="The image repository and optionally a tag in the 'repository:tag' format.")
+        c.argument('image_name', options_list=['--image', '-t'], help="The image repository and optionally a tag in the 'repository:tag' format.")
```
Comment (djyou) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py:87]: I thought you mentioned docker's convention is to use `--tag` and `-t`?

**Review by djyou — APPROVED** [review]
Just some minor points.

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py (hunk 2 above)
Comment (northtyphoon) [reply src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py:87]: Yes. -t is used in most docker build commands as we can see. As you mentioned, --tag is already used in existing acr commands as a different definition. So I think it's a good tradeoff here.

**Pull request merged**